src/features/groups/list/category-tab/index.styled.ts

```diff
@@ -0,0 +1,37 @@
+import styled, { css } from 'styled-components';
+
+export const CategoryTabWrapper = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+`;
+
+interface ICategoryTabProps {
+  active?: boolean;
+}
+
+export const CategoryText = styled.div<ICategoryTabProps>`
+  display: flex;
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
```

Comment:
please consider using `em` or `rem` instream of hardcoded pixels

Reply:
We use 'px' because of Figma css show sizes in 'px', but if it refers only to font size, than ok, it should not be difficult to use 'em' or 'rem' :)

Reply:
I don't think there is sense to use `em` only for `font-size`. In case of using `em` we need to use it everywhere, for paddings, margins and so on.. Don't you agree?

I don't think we need it. But we can discuss on some call :)